phpstan.neon.dist

```diff
@@ -6,14 +6,14 @@ parameters:
         - src
     ignoreErrors:
         -
-            message: "#^Strict comparison using \\!\\=\\= between null and null will always evaluate to false\\.$#"
+            message: "#^Cannot call method createStream\\(\\) on Psr\\\\Http\\\\Message\\\\StreamFactoryInterface\\|null\\.$#"
```

Comment:
this is git diff being weird. i delete the strict comparison thing for HttpMethodsClient and add the one for the RoundRobinClientPool